executor/executor_test.h

```diff
@@ -5,11 +5,22 @@
 #include <sys/mman.h>
 #include <unistd.h>
 
+#ifdef __unix__
+#include <sys/prctl.h>
+#endif
+
 static uint64 kernel_text_start = 0xc0dec0dec0000000;
 static uint64 kernel_text_mask = 0xffffff;
 
 static void os_init(int argc, char** argv, void* data, size_t data_size)
 {
+#ifdef __unix__
```

Comment:
Won't __unix__ include *BSD which may not have PR_SET_PDEATHSIG?

Reply:
Do you know a reliable way to distinguish Linux from *BSD in the C++ source?

Reply:
`#if GOOS_linux`

Reply:
But we have GOOS_test, don't we?

Reply:
Right. There should be `__linux__`.

Reply:
Updated to __linux__